hatchways/team-wintergreen
# Issue 31 in manage bookings

### What this PR does (required):
- Integration booking system
-- make a booking from the profile page
-- get all bookings for the sitter
-- update booking (accept or decline)

### Screenshots / Videos:
![image](https://user-images.githubusercontent.com/32376967/154144566-ef4f53cb-6c79-4181-a1fd-3dd40f298e66.png)
![image](https://user-images.githubusercontent.com/32376967/154144710-90be3074-f4c7-43c5-8065-c4f82b8858c0.png)


### Any information needed to test this feature (required):
- make a new booking on the profile page
- see all booking Infos on the booking page
- accept or decline a booking and it will save after logging out

### Any issues with the current functionality (optional):
- based on the mock-up, there does not have status info for the next booking, therefore I don't add any function for the setting button for the next booking slot.
- did not implement validation for start date and end date yet.
